packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-visitor.test.ts

```diff
@@ -317,6 +317,31 @@ describe('AppSyncModelVisitor', () => {
       expect(commentsField).not.toContain('post');
       expect(commentsField).toContain('postCommentsId'); // because of connection from Post.comments
     });
+
+    it('should not generate projectTeamId connection field for hasOne directive', () => {
```

Comment:
The name is misleading here as we are actually asserting that `projectTeamId` exists.